platform/marvell-armhf/sonic-platform-nokia/7215/scripts/cpu_wdt.py

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/python
+
+from sonic_platform.chassis import Chassis
+import time
+import os
+import signal
+import sys
+
+TIMEOUT=170
+KEEPALIVE=55
+
+time.sleep(60)
+chassis = Chassis()
+watchdog = chassis.get_watchdog()
+
+def stopWdtService(signal, frame):
+    watchdog._disablewatchdog()
+    print(" Watchdog disarmed, CPUWDT has been disabled ")
+    sys.exit()
+
+def main():
+
+    signal.signal(signal.SIGHUP, signal.SIG_IGN)
+    signal.signal(signal.SIGINT, stopWdtService)
+    signal.signal(signal.SIGTERM, stopWdtService)
+    
+    watchdog.arm(TIMEOUT)
```

Comment:
How about add syslog here when enabling the watch dog? Then we can know when the watchdog start work